### PR TITLE
fix assert

### PR DESCRIPTION
**Pull request by maxime-guichard**

### CODE REVIEW

**Review by Erdnaxela45 — DISMISSED** [review]

File: src/Entity/assert [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    /**
+     * @Assert\Notblank(message="Le champ Email est obligatoire")
+     * @Assert\Email(message = "L'adresse '{{ value }}' n'est pas une adresse mail valide.")
+     * @Assert\Length(max="100", maxMessage="L'adresse mail ne doit pas dépasser {{ limit }} caractères")
+     */
+    private ?string $email;
+
+    /**
+     * @Assert\Notblank(message="Le champ Message est obligatoire")
+     */
```
Comment (LucyMerlier) [inline src/Entity/assert:22]: what is this file doing here?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by Erdnaxela45 — DISMISSED** [review]

File: templates/component/_navbar.html.twig [hunk 2]
```diff
@@ -18,7 +18,7 @@
                 <a class="nav-link ms-5 me-5 active" href="#">Connexion</a>
             </li>
             <li class="nav-item">
-                <a class="nav-link ms-5 me-5 active" href="#">Contact</a>
+                <a class="nav-link ms-5 me-5 active" href="/contact/index">Contact</a>
```
Comment (LucyMerlier) [inline templates/component/_navbar.html.twig:21]: never put urls in hrefs, always use twig function path

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by JuMn88 — DISMISSED** [review]

**Review by LucyMerlier — DISMISSED** [review]

**Review by Erdnaxela45 — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**